modules/drivers/sparksql/test/metabase/test/data/sparksql.clj

```diff
@@ -83,24 +82,20 @@
     ((get-method ddl/insert-rows-honeysql-form :sql/test-extensions) driver table-identifier rows)))
 
 (defmethod load-data/do-insert! :sparksql
-  [driver spec table-identifier row-or-rows]
+  [driver ^java.sql.Connection conn table-identifier row-or-rows]
   (let [statements (ddl/insert-rows-ddl-statements driver table-identifier row-or-rows)]
-    (sql-jdbc.execute/do-with-connection-with-options
-     driver
-     spec
-     {:write? true}
-     (fn [^java.sql.Connection conn]
-       (try
-         (.setAutoCommit conn false)
```

Comment:
Auto-commit false here doesn't make sense, databases generally get pretty fussy if you try to do DDL stuff inside transactions. Changed to auto-commit = true. If data loading fails it destroys the whole DB anyway.